benchmarks/src/benchmark_serialization_twitter.cpp

```diff
@@ -27,12 +25,30 @@ struct User {
     int64_t followers_count;
     int64_t friends_count;
     int64_t statuses_count;
+
+    bool operator==(const User &other) const {
```

Comment:
This is fine, but I think it can be simplified.

Reply:
What do you have in mind here?